src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
**Enum** is usually written using UPPER_CASE and UNDERSCORES('_')

Exaample: BLACK, WHITE, ...
ToDo: use UPPER_CASE and UNDERSCORES, if you need them in lower case, you can use method `.toLowerCase()` on the String color value
OR
If you want can add variables to enums [example](.toLowerCase())